blog/2023-01-18-introducing-netdata-paid-plans.md

```diff
@@ -0,0 +1,95 @@
+---
+slug: introducing-netdata-paid-plans
+title: Introducing Netdata Paid Plans
+description: All Netdata functionality is and will be available for free in the Free Community Plan. Paid tiers include features targeted for managers, and administrators.
+authors: team
+tags: [pricing, free-plan, paid-subscriptions]
+keywords: [pricing, free plan, paid subscriptions, monitoring, troubleshoot]
+image: https://user-images.githubusercontent.com/24860547/212873204-5082b40d-7be0-4a08-ae72-0895b840837b.png
+
+---
+
+Read more about Netdata introducing paid subscriptions. All Netdata functionality is and will be available for free in the Free Community Plan. Paid tiers include features targeted for managers, and administrators.
+
+<!--truncate-->
+## Introducing Netdata paid subscriptions
+
+At Netdata we believe that as the digitization of our world matures, it is mandatory for all of us to have free and unrestricted access to high quality monitoring solutions. Concepts like instant deployment, real-time access to unlimited high resolution data, fully automated and standardized dashboards and alarms, and machine learning assisted troubleshooting, will eventually allow all of us to focus on how to bring better results for our businesses and lives, abstracting the complexity and flattening the learning curve of using efficiently, incrementally more advanced software solutions in our software stacks.
```

Comment:
At Netdata we believe that as the digitization of our world matures, it is mandatory for all of us to have free and unrestricted access to high quality monitoring solutions. Concepts like instant deployment, real-time access to unlimited high resolution data, fully automated and standardized dashboards and alarms, and machine learning assisted troubleshooting, will eventually allow all of us to focus on how to bring better results for our businesses and lives. Our aim is to abstract the complexity and flatten the learning curve of using efficiently, incrementally more advanced software solutions in our software stacks.